src/components/Calendar/Header/Topbar/Topbar.tsx

```diff
@@ -10,23 +10,43 @@ const Topbar: FC<TopbarProps> = ({ width }) => {
   const { topbar } = useLanguage();
   const {
     data,
+    config,
     handleGoNext,
     handleGoPrev,
     handleGoToday,
     zoomIn,
     zoomOut,
     isNextZoom,
     isPrevZoom,
-    handleFilterData
+    handleFilterData,
+    onClearFilterData
   } = useCalendar();
   const { colors } = useTheme();
+  const { filterButtonState = -1 } = config;
+
+  const handleClearData: MouseEventHandler<HTMLButtonElement> = (event) => {
```

Comment:
Can we change the name to `clearFilters`? Clear data sounds like we are going to remove resources

Reply:
Changed in [4650c95](https://github.com/Bitnoise/react-scheduler/pull/27/commits/4650c95a85110a5d6eccc080032655f298b67538)